log4j-1.2-api/src/main/java/org/apache/log4j/builders/appender/RewriteAppenderBuilder.java

```diff
@@ -50,7 +50,7 @@
 
 
 /**
- * Build an Asynch Appender
+ * Build an Async Appender
```

Comment:
This seems to be the RewriteAppender so this comment was completely wrong.

Reply:
HI @rgoers 
Agree. changed.
TY